### PR TITLE
Created index.html

### PR DESCRIPTION
**Pull request by zeel1998**
Warm welcome by Zarna Joshi

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+
+<html>
+<body>
+<h1>Hi GUYS</h1>
+</body>
+</html>
```
Comment (JayVyas01) [inline index.html:11]: Where is the output of this?

**Review by JayVyas01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (zeel1998) [reply index.html:11]: CommentOutput

Comment (JayVyas01) [reply index.html:11]: where is the title of the html page

Comment (zeel1998) [reply index.html:11]: CommentOutput
Joi le joi le

**Review by JayVyas01 — APPROVED** [review]

**Review by KaranJoshi03 — APPROVED** [review]

**Pull request merged**